src/main/java/com/conveyal/gtfs/error/RangeError.java

```diff
@@ -9,7 +11,7 @@
     final double min, max, actual;
 
     public RangeError(String file, long line, String field, double min, double max, double actual) {
-        super(file, line, field);
+        super(file, line, field, Priority.MEDIUM);
```

Comment:
Maybe low? I see this often (maybe with stop_times.pickup_type?)
```suggestion
        super(file, line, field, Priority.LOW);
```